src/main/java/com/kaziamyr/onlinebookstore/model/ShoppingCart.java

```diff
@@ -0,0 +1,37 @@
+package com.kaziamyr.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+@Entity
+@SQLDelete(sql = "UPDATE shopping_carts SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
+@Table(name = "shopping_carts")
+@Data
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(nullable = false)
+    private User user;
+    @OneToMany(mappedBy = "shoppingCart")
```

Comment:
Please specify join column

Reply:
if for example, we will have 2 or more shopping carts will all of them be equal for the same user?